docs/ref/models/querysets.txt

```diff
@@ -2562,7 +2562,7 @@ Case-insensitive starts-with.
 
 Example::
 
-    Entry.objects.filter(headline__istartswith='will')
+    Entry.objects.filter(headline__istartswith='Will')
```

Comment:
I see what you mean.  I was following the convention in the documentation for `startswith`.  Would you prefer that it match `iendswith`?